### ISSUE TITLE
Use black font on mobi

### ISSUE THREAD

**Issue by SamyPesse** [GitbookIO/gitbook#784]
Mobi On Kindle Voyage's font color is not black but grey.

**Comment by 9beach** [comment]
Are there any work-rounds?

**Comment by kaizora** [comment]
+1. All my books from gitbook are hard to read on a KindlePW due to this grey font...

**Comment by youxiachai** [comment]
current I use calibre epub -> azw3 solve this problem

look & feel -> extra css use
```
.page {
    -ms-text-size-adjust: 100%;
    -webkit-text-size-adjust: 100%;
    color: #000; // Note default is #333 It is kpw show grey reason
    display: block;
    font-family: sans-serif;
    line-height: 1.6;
    overflow: hidden;
    word-wrap: break-word
    }
```

**Comment by SamyPesse** [comment]
Fixed by #904
